cg/meta/deliver_ticket.py

```diff
@@ -0,0 +1,142 @@
+"""Module for deliver and rsync customer inbox on hasta to customer inbox on caesar"""
+import logging
+import os
+import re
+import subprocess
+from pathlib import Path
+import shutil
+from typing import List
+
+from cg.exc import CgError
+from cg.meta.meta import MetaAPI
+from cg.models.cg_config import CGConfig
+from cg.store import models
+
+
+LOG = logging.getLogger(__name__)
+PREFIX_TO_CONCATENATE = ["MWG", "MWL", "MWM", "MWR", "MWX"]
+
+
+class DeliverTicketAPI(MetaAPI):
+    def __init__(self, config: CGConfig):
+        super().__init__(config)
+        self.delivery_path: str = config.delivery_path
+
+    def get_all_cases_from_ticket(self, ticket_id: int) -> list:
+        cases: List[models.Family] = self.status_db.get_cases_from_ticket(ticket_id=ticket_id).all()
+        return cases
+
+    def get_inbox_path(self, ticket_id: int) -> str:
+        cases = self.get_all_cases_from_ticket(ticket_id=ticket_id)
+        if not cases:
+            LOG.warning("Could not find any cases for ticket_id %s", ticket_id)
+            raise CgError()
```

Comment:
Try to be more specific. Use an exception that is more relevant to the problem and include a message